### ISSUE TITLE
V panic:  converting jdk_version "9" to semantic version.

### ISSUE THREAD

**Issue by rcqls** [vlang/vab#63]
To make the current code for my macos install working, I simply commented this lines in `vab.v`:

```
        // jdk_semantic_version := semver.from(jdk_version) or {
	// 	panic(@MOD + '.' + @FN + ':' + @LINE +
	// 		' error converting jdk_version "$jdk_version" to semantic version.\nsemver: ' + err)
	// }
	// if !jdk_semantic_version.ge(semver.build(1, 8, 0)) { // NOTE When did this break:.satisfies('1.8.*') ???
	// 	// Some Android tools like `sdkmanager` in cmdline-tools;1.0 only worked with Java 8 JDK (1.8.x).
	// 	// (Absolute mess, yes)
	// 	eprintln('Java JDK version $jdk_version is not supported')
	// 	eprintln('Please install Java >= 8 JDK or provide a valid path via JAVA_HOME')
	// 	exit(1)
	// }
```
Otherwise, it generates the panic error:
`V panic: main.validate_env:393 error converting jdk_version "9" to semantic version.`

```
vab doctor --api 29
vab
	Version 0.2.1 a16801a

	Path "/Users/rcqls/vlang/vab"
Java
	JDK
		Version 9
		Path "/Library/Java/JavaVirtualMachines/adoptopenjdk-9.jdk/Contents/Home"
Android
	ENV
		sdkmanager "/Users/rcqls/.cache/v/android/cmdline-tools/tools/bin/sdkmanager"
		sdkmanager.version "4.0.1"
		Managable: true
	SDK
		Path "/Users/rcqls/Library/Android/sdk"
		Writable true
	NDK
		Version 22.0.7026061
		Path "/Users/rcqls/Library/Android/sdk/ndk"
		Side-by-side true
	Build
		API 29
		Build-tools 30.0.3
Product
	Name "V Test App"
	Package "io.v.android"
	Output "v_test_app.apk"
V
	Version 0.2.2 db0fc8f
	Path "/Users/rcqls/vlang/v"

	OS: macos, macOS, 11.1, 20C69
	Processor: 4 cpus, 64bit, little endian, Intel(R) Core(TM) m7-6Y75 CPU @ 1.20GHz
	CC version: Apple clang version 12.0.0 (clang-1200.0.32.29)

	getwd: /Users/rcqls/vlang/apk
	vmodules: /Users/rcqls/.vmodules
	vroot: /Users/rcqls/vlang/v
	vexe: /Users/rcqls/vlang/v/v
	vexe mtime: 2021-02-06 21:04:32
	is vroot writable: true
	is vmodules writable: true
	V full version: V 0.2.2 db0fc8f

	Git version: git version 2.30.0
	Git vroot status: weekly.2021.5-115-gdb0fc8fb-dirty (2 commit(s) behind V master)
	.git/config present: true
	thirdparty/tcc status: thirdparty-macos-amd64 689c8a02
```

**Comment by Larpon** [comment]
@rcqls - thanks for the report - just reopen if this didn't fix it!

**Comment by rcqls** [comment]
@Larpon Thanks for the fix., However, it seems that it  needs `version += '.0.0'` instead of `version += '.0'` to really have a proper semantic version.

**Comment by Larpon** [comment]
Yup, thanks! Fixed in #65
